venia-integration-tests/src/fields/accountMenu/index.js

```diff
@@ -0,0 +1,3 @@
+export const accountMenuItemsField = 'div[data-id="account-menu-items"]';
```

Comment:
Consider using a slightly more specific name for this data attribute to avoid collisions.

Reply:
I added this as a sample. We don't have `data-ids` in our code base yet. These are just dummy ones. When each dev decides to add new tests they can update these files accordingly.